src/sagas/pushNotification.js

```diff
@@ -334,7 +392,8 @@ const hasPostNotificationAuthorization = async () => {
  * Opens the app settings screen where the user can enable the notification settings.
  */
 const openAppSettings = async () => {
-  if (Platform.OS === 'android') {
+  if (Platform.OS === 'android'
+      || Platform.OS === 'ios') {
```

Comment:
Can we just remove this `if`?

Reply:
Yes.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-mobile/pull/268/commits/67ee4e63d11736cd78932e55234c1a988841a66d